README.md

```diff
@@ -29,9 +29,11 @@ Why?
 
 ## Usage
 
-To clone the repository and run this website on your local machine, [install Jekyll](https://jekyllrb.com/docs/installation/) for your OS and type the following commands:
-
-    $ sudo gem install jekyll bundler
+To clone the repository and run this website on your local machine, install
```

Comment:
Reading the first line, creates a thought that for cloning this repository we need to install `asdf-vm`. Possibly you can divide this into two sections - `asdf-vm & ruby installation` and `cloning and running`

Reply:
@KVGarg I agree with you but in the original README.md they asked to install Jekyll with the first line. So maybe they thought it won't create the confusion. What do you think?

Reply:
In original README.md also, it is creating a confusion too. Now, also it's ok but it is creating some confusion. Don't know about others. Aren't you getting confused ?

Reply:
yeah, it does creates the confusion. i will change it.